tests/integration/features/stable.feature

```diff
@@ -684,28 +684,6 @@ Feature: Testing the update scenario of stable releases
     And the installation mtime is "31"
     When The request is sent
     Then The response is non-empty
-    And Update to version "20.0.12.1" is available
-    And URL to download is "https://download.nextcloud.com/server/releases/nextcloud-20.0.12.zip"
-    And URL to documentation is "https://docs.nextcloud.com/server/20/admin_manual/maintenance/upgrade.html"
-    And EOL is set to "0"
-    And The signature is
-    """
-    MUo2UnLsQwjwkAQidakJchr223IymVUh9/lLzHMPqCwNIKjcKI7AYgmuaXxJPQQv
-    JwlEzlGCF3jayDFaGsf83yZLHYEcs0EBG62vYfps7p2tpkiF1R+Kcu4KnVWT040D
-    w4lSUhBc4hlTtJF+5Wur8frkfKuhiVARDjKtNhchmeKj7mi4PoeaP5zsZfTo+azP
-    fJc3gJ6PhOFQBLWgoKsE9I6fjhvMgPLH5VjA3NAvwdP7/7mA2Y1982OcuImZeZ2H
-    KwzuCZgZ1cw8k/ii8cBk+K+bdeiZxV6S4X40zFIeYDUeqkAghoHWWyaOWzGy9WGp
-    PIjDHj/2nZQr50sIrdVWwg==
-    """
-
-
-  Scenario: Updating the Nextcloud 20.0.11 on the stable channel
-    Given There is a release with channel "stable"
-    And The received version is "20.0.11.1"
-    And The received PHP version is "7.3.0"
-    And the installation mtime is "11"
-    When The request is sent
-    Then The response is non-empty
     And Update to version "21.0.4.1" is available
```

Comment:
20 < 20.0.12 must go to latest maintenance release first

Reply:
![Screenshot_20210811_105232](https://user-images.githubusercontent.com/2184312/128999663-7416e2ac-3d52-4926-bb23-b44f59c2354f.png)